.gitignore

```diff
@@ -3,6 +3,8 @@
 *.pdb
 *.mo
 *.exe
+.kdev4/
```

Comment:
What is this for?

Reply:
This is for KDevelop, but I can remove it if you prefer not to have anything IDE-specific in the .gitignore.

Reply:
I'm okay with it being added, but it should definitely be in a separate section from actual build files and labeled appropriately ("IDE-specific" maybe).

Reply:
Done.

Reply:
I generally suggest adding IDE-specific paths to your global gitignore, as they are not related to the code itself and different per user.